saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/data/CLEFImageReader.java

```diff
@@ -321,26 +321,6 @@ private void getTrainingImages() throws IOException {
                 trainingData.add(Integer.toString(imageID));
             }
         }
-
-        String crossValidationImage = path + "/validation.mat";
-        d = new File(trainImage);
-
-        if (!d.exists()) {
-            throw new IOException(crossValidationImage + " does not exist!");
-        }
-
-        MatFileReader matValidaitonreader = new MatFileReader(crossValidationImage);
```

Comment:
@umarmanzoor In the previous PR, we had some unconventional namings, please correct them as well.